packages/website/pages/404.js

```diff
@@ -1,14 +1,21 @@
-// ===================================================================== Imports
+import { useEffect } from 'react';
+
 import GeneralPageData from '../content/pages/general.json';
 import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
+import countly from '../lib/countly';
 
-// ===================================================================== Exports
 const blocks = GeneralPageData.error_404.sections;
 
-const Custom404 = () => (
-  <main className="page-404">
-    <BlockBuilder id="error_section-1" subsections={blocks} />
-  </main>
-);
+export default function Custom404() {
+  useEffect(() => {
+    countly.trackEvent(countly.events.NOT_FOUND, {
+      path: '/404',
```

Comment:
Might it be useful to get this value from `useRouter()`? https://nextjs.org/docs/api-reference/next/router#userouter